### PR TITLE
Set up a 'Hello Microverse Project'

### PR DESCRIPTION
**Pull request by kifle23**
Added MIT Licence,.hintrc,stylelintrc.json based on my first pull request feedback

### CODE REVIEW

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @kifle23 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

212385281-4797c58a-d358-423a-aa7c-832c22203865


 ## Highlights
✔No linter errors
✔Good efforts

 ## Required Changes ♻️

_Great job implementing some of requested changes by previous reviewer , but you still need to improve your README to make it professional , Please put again all sections from microverse template only faq section is not required  if you don't have content for a section you can mention that it will be added soon , and finally remove all default text after customizing with your own content .
_Also please always use a PR title and summary that describes the work done in the feature 
_kindly fix linter errors before requesting next review  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,135 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it some ⭐️s
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (shella12) [inline README.md]: - [ ] Great job on adding missing sections but according to the content table there are still two of the sections missing. Please make sure to include `future features` section inlist 2-3 future features of the project and `Acknowlegment` section to give credit to the people who's content has been used in the project

File: README.md [hunk 2]
```diff
@@ -1 +1,135 @@
...
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (shella12) [inline README.md]: - [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @kifle23 , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- All linters passed ✅  
- Descriptive pull request ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I suggest using a more descriptive name for your feature branch instead of `newBranch`. Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section’) 


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kifle23** [comment]
I have changed the branch name from newBranch to add-feature but this automatically closes my pull request. I had to roll it back to preserve the pull request but I've learned the lesson. Thank you!

File: README.md (hunk 2 above)
Comment (kifle23) [reply README.md]: I have changed the branch name from newBranch to add-feature but this automatically closes my pull request. I had to roll it back to preserve the pull request but I've learned the lesson. Thank you!

**Review by jasonfelice — APPROVED** [review]
H  @kifle23, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**